tutorials/time-freq/ssvep.py

```diff
@@ -0,0 +1,359 @@
+"""
+==================================
+Basic analysis of an SSVEP dataset
+==================================
+
+Example script to compute frequency spectrum and extract snr of a target frequency
+
+We use a simple example dataset with frequency tagged visual stimulation (a.k.a.
+steady state visually evoked potentials - SSVEP):
+
+N=2 participants observed checkerboards patterns inverting with a constant frequency
+of either 12Hz of 15Hz. 10 trials of 30s length each. 32ch wet EEG was recorded.
+
+Data format: BrainVision .eeg/.vhdr/.vmrk files organized according to BIDS standard.
+
+Data can be downloaded at https://osf.io/7ne6y/
+"""  # noqa: E501
+# Authors: Dominik Welke <dominik.welke@ae.mpg.de>
+#          Evgenii Kalenkovich <e.kalenkovich@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+import mne
+import numpy as np
+from mne_bids import make_bids_basename, read_raw_bids
+from scipy.stats import ttest_rel, ttest_ind
+
+###############################################################################
+# Set parameters
+# Load eeg trigger structure / event_id
+event_id = {
+    '12hz': 10001,
+    '15hz': 10002
+}
+
+datafile = "tja"
+bids_root = "./data/"
+bids_filename = make_bids_basename(
+    subject='02',
+    session='01', task='ssvep'
+) + '_eeg.vhdr'
+
+###############################################################################
+#  Load raw data
+raw = read_raw_bids(bids_filename, bids_root)
+raw.load_data()
+
+###############################################################################
+# Minimal preprocessing
+# ---------------------
+#
+# Due to a generally high SNR in freqeuncy tagging, typical preprocessing steps
+# are considered optional. this doesnt mean, that a proper cleaning would not
+# increase your signal quality!
+#
+# Raw data comes with FCz recording reference, so we will apply common-average
+# rereferencing.
+
+###############################################################################
+# Set montage.
```

Comment:
can all these be formatted as subheadings? or at least in bold font?

Reply:
Done. Now I know how it works in sphinx:

```
Section headers (ref) are created by underlining (and optionally overlining) the section title with a punctuation character, at least as long as the text:

=================
This is a heading
=================
Normally, there are no heading levels assigned to certain characters as the structure is determined from the succession of headings. However, this convention is used in Python’s Style Guide for documenting which you may follow:

# with overline, for parts
> * with overline, for chapters
> =, for sections
> -, for subsections
> ^, for subsubsections
> ", for paragraphs
```